internal/persistence/sql/relationtuples.go

```diff
@@ -176,6 +191,9 @@ func (p *Persister) whereSubject(ctx context.Context, q *pop.Query, sub relation
 }
 
 func (p *Persister) whereQuery(ctx context.Context, q *pop.Query, rq *relationtuple.RelationQuery) error {
+	ctx, span := p.d.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.whereQuery")
```

Comment:
Where all are we adding tracing? I'm genuinely curious, as I haven't worked much with tracing before. This function will not perform any DB or IO actions itself but just create the query. Do we also trace these intermediate functions, or is it more for DB interactions?

Reply:
Yes, we only want to trace DB/IO or other "slow" operations to help pin-point potential performance bottlenecks. In this case — I must admit — I didn't pay close attention to what the function was actually doing and simply created a child span. While it does no harm existing, I suppose I'll go ahead and remove it for cleanliness' sake. :)